app/views/command-line.html

```diff
@@ -38,7 +38,12 @@ <h1 id="cli">Command Line Tools</h1>
                   <p>
                     After downloading and installing it, you can start by logging in using<span ng-if="sessionToken"> this current session token</span>:
                     <div class="code prettyprint ng-binding" ng-if="sessionToken">
-                      oc login {{loginBaseURL}} --token=<span ng-show="showSessionToken">{{sessionToken}}</span><a href="#" ng-click="toggleShowSessionToken()" ng-show="!showSessionToken">...click to show token...</a>
+                      oc login {{loginBaseURL}} --token=<span ng-show="showSessionToken">{{sessionToken}}</span><span ng-show="!showSessionToken">&lt;hidden&gt;</span>
+                      <span ng-show="!showSessionToken">
+                        <a href="#" ng-click="toggleShowSessionToken()">Show token</a>
+                        <span class="action-divider">|</span>
+                        <a href="#" id="{{copySessionTokenId}}" ng-click="copySessionToken(this)" title="Copy command to clipboard" data-toggle="tooltip">Copy command to clipboard</a>
```

Comment:
Hide this if it's iOS? There's an `IS_IOS` constant you can use in the controller to set a scope var.